dwq/dwqw.py

```diff
@@ -10,7 +10,7 @@
 import traceback
 import multiprocessing
 import shutil
-from subprocess import run, PIPE, STDOUT, TimeoutExpired
+from subprocess import run, PIPE, STDOUT, TimeoutExpired, CalledProcessError
```

Comment:
Ah, I just found that through other means (staring at logs), it's in master already.

Reply:
Doesn't seem to create a merge conflict, though. (I can rebase and eliminate the commit if you prefer, but I'd just leave it as is if it doesn't disturb review of the rest of this PR)